elide-core/src/main/java/com/yahoo/elide/jsonapi/models/Relationship.java

```diff
@@ -80,6 +81,8 @@ public Relationship(@JsonProperty("links") Map<String, String> links,
                     res.add(resource.toPersistentResource(requestScope));
                 } catch (ForbiddenAccessException e) {
                     //skip resource
+                } catch (java.lang.NullPointerException e) {
+                    throw new InvalidRelationException(resource.getType());
```

Comment:
What causes this NPE? Wouldn't it be better to throw this exceptions instead of an NPE?